apps/saleor-app-checkout/package.json

```diff
@@ -9,6 +9,7 @@
     "build": "npm run i18n && next build",
     "start": "next start -p 3001",
     "lint": "next lint",
+    "lint:staged": "next lint --fix",
```

Comment:
Why are we using `--fix` in `lint:staged` in saleor-app-checkout and not inside `lint` like in other packages?

For example: [`packages/checkout-common/package.json`](https://github.com/saleor/react-storefront/pull/428/files#diff-e0d1fd5d7fc07043ea55916c748c9948b090bb0c052116e3f3759cb11ce3e27f)

Reply:
We should be doing `--fix` everywhere